service/package/rubygem-agama-yast.changes

```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Fri Sep 27 09:23:40 UTC 2024 - Imobach Gonzalez Sosa <igonzalezsosa@suse.com>
+
+- Handle language change in users and storage (gh#agama-project/agama#1642).
```

Comment:
NP: well ... instead of "language change" i'd use something like "switching UI language" ... to make obvious that the fix addresses switching of the language the UI uses instead of default EN